cirq/ops/moment_test.py

```diff
@@ -579,3 +579,16 @@ def test_commutes():
     assert not cirq.commutes(moment, cirq.H(a))
     assert not cirq.commutes(moment, cirq.H(b))
     assert not cirq.commutes(moment, cirq.X(c))
+
+
```

Comment:
we should add a test method that tests the deprecation logic as well, ensures the right warning message is logged.

Reply:
added a test for the deprecation logic.